app-backend/common/src/main/scala/com/rasterfoundry/common/color/ColorCorrect.scala

```diff
@@ -168,10 +166,26 @@ object ColorCorrect extends LazyLogging {
       val iMaxMin: Array[(Int, Int)] = Array.ofDim(3)
       cfor(0)(_ < _rgbHist.length, _ + 1) { index =>
         val hst = _rgbHist(index)
+        val statsOption = hst.statistics()
         val imin = hst.minValue().map(_.toInt).getOrElse(0)
         val imax = hst.maxValue().map(_.toInt).getOrElse(255)
-        logger.trace(s"Histogram Min/Max: ${imin}/${imax}")
-        iMaxMin(index) = (imin, imax)
+
+        // if data is a byte raster (0, 255) don't do anything
+        // else if stats are available, clip assuming a normal distribution
+        // else use the histogram's min/max
+        (imin, imax, statsOption) match {
+          case (0, 255, _) => iMaxMin(index) = (0, 255)
+          case (_, _, Some(stats)) =>
+            // assuming a normal distribution, clips 2nd and 98th percentiles of values
```

Comment:
I'd err towards squashing the values rather than clipping - this will result in stuff like bad sensor pixels or exposure looking like nodata, when they really shouldn't be treated as such.